quic-forward-proxy/src/util.rs

```diff
@@ -0,0 +1,41 @@
+#![allow(dead_code)]
+use solana_sdk::signature::Keypair;
+use std::env;
+use std::future::Future;
+use std::time::Duration;
+
+use tokio::time::Timeout;
+
+pub type AnyhowJoinHandle = tokio::task::JoinHandle<anyhow::Result<()>>;
+pub const FALLBACK_TIMEOUT: Duration = Duration::from_secs(5);
+
+pub fn timeout_fallback<F>(future: F) -> Timeout<F>
+where
+    F: Future,
+{
+    tokio::time::timeout(FALLBACK_TIMEOUT, future)
+}
+
+// note this is duplicated from lite-rpc module
```

Comment:
should move the code where both of the libraries can use.

Reply:
yes but where - new crate?

Reply:
May be core.

Reply:
done